examples/basics/parameters/parameter-passing.js

```diff
@@ -0,0 +1,69 @@
+const createSession = require('../../session');
+
+// inline script with a simple table
+const script = `
+exampletable:
+load * Inline [
+dim, val
+foo, 23
+bar, 42
+c, 54
+d, 21
+];
+`;
+
+const session = createSession();
+
+/**
+ * Write the tabelContent to console.
+ * @param {*} tabelContent
+ */
+function print(tableContent) {
+  for (let i = 0; i < tableContent.length; i += 1) {
+    console.log(`dim: ${tableContent[i].qValue[0].qText} val: ${tableContent[i].qValue[1].qText}`);
+  }
+}
+
+// local variable to store the generated sessionapp
+let sessionApp;
+
+session.open()
+  .then(global => global.createSessionApp())
+  .then((app) => {
+    sessionApp = app;
+    return app.setScript(script);
+  })
+  .then(() => sessionApp.doReload())
+  .then(() =>
+    /**
+     * The getTableData function returns the data from a specified table.
+     *
+     * The first example shows how to pass the parameter by position. By this way you pass
+     * parameters by position in the order defined by the QIX method. Hereby the order
+     * of the parameters is important
```

Comment:
Missing .

Actually, I think the last sentence is a bit redundant since it's already stated that parameters should be passed in the order defined bu the QIX method.